### PR TITLE
[SMP-1646] SMP 0.8.0 external self-managed Timescaledb tutorial

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.8.0 external self-managed Timescaledb tutorial

For reviewers, a preview is available:

[Configure an external self-managed TimescaleDB](https://64c53e54b41ae10634ade972--harness-developer.netlify.app/tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb)

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]
LGTM, based on the technical details provided

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+
+4. Add the TimescaleDB third party repository.
+
+   Debian:
+   
+   ```
+   echo "deb https://packagecloud.io/timescale/timescaledb/debian/ $(lsb_release -c -s) main" | sudo tee /etc/apt/sources.list.d/timescaledb.list
+   ```
+
+   Ubuntu 21.10 an later:
```
Comment (arya-harness) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:113]: Ubuntu 21.10 and later:

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+
+   ```
+   psql (13.11 (Debian 13.11-1.pgdg110+1))
+   Type "help" for help
+   ```
+
+5. Set the password for the postgres user.
+
+   ```
+   \password postgres
```
Comment (arya-harness) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:184]:
```
   \password postgres
   ```
   It is causing an extra line

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+   ```
+   cd /var/lib/postgresql/13
+   mv main main.org
+   mkdir main
+   ```
+
+3. Restore from the base backup, using the IP address of the primary database and the replication username.
+
+   ```
+   pg_basebackup -h <primary> -p 5432 -U reptest -D /var/lib/postgresql/13/main/ -Fp -Xs -R
```
Comment (arya-harness) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:380]: We should change it to 
pg_basebackup -h <PRIMARY_IP>

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+2. Change the host in your Harness instance, and upgrade your Helm charts.
+
+### Recover the former primary
+
+You can recover the former primary VM after you promote the standby/secondary.
+
+Replica promotion affects primary and secondary use in the following way:
+
+   old primary ↔︎ new secondary 
+   new primary ↔︎ old secondary ↔︎ old standby
```
Comment (arya-harness) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:506]: the formatting looks a bit off, can we put a new line between these terminologies?

**Comment by arya-harness** [comment]
Thanks @brian-f-harness , the document looks good.

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 5]
```diff
@@ -0,0 +1,556 @@
...
+
+- Maintenance overhead: VMs require regular maintenance, including updates, backups, and monitoring, which adds overhead compared to a native TimescaleDB installation.
+
+## Hardware requirements
+
+Harness recommends a PostgreSQL three-member replica set configuration with the following minimum hardware:
+
+- Three nodes
+- Four CPU (3*4 = 12 CPU)
+- 24GB RAM (3*16 = 48GB RAM)
```
Comment (arya-harness) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:39]: The multiplication is a bit off, @pratmit are we aiming for 16GB per replica or 24GB(recommended by Sreehari for very operational based system)?

Comment (brian-f-harness) [reply tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:39]: @arya-harness Thanks for catching this. I updated it with the value from the doc: 24GB RAM (3*24 = 72GB RAM total)

**Review by arya-harness — APPROVED** [review]

File: tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md [hunk 6]
```diff
@@ -0,0 +1,558 @@
...
+External database setup requires the following software:
+
+- Supported OS: TimescaleDB supports Debian 11, Ubuntu, and CentOS
+
+
+## Network requirements
+
+Ensure the following:
+
+- Set allowlisting of VMs so each VM can send traffic. Add the source using TimescaleDB, for example, the Kubernetes cluster service range to your allowlist.
```
Comment (StrawberryPad) [inline tutorials/self-managed-enterprise-edition/use-an-external-sm-timescaledb.md:53]: "allowlist" not used consistently in this topic. You also have "allow list", and the other tutorial uses "allowlist". I think both are fine, but need to choose one of them.

**Review by StrawberryPad — APPROVED** [review]

**Pull request merged**